resources/web/template.html

```diff
@@ -161,1115 +101,314 @@
+          $('.mobile-navigation, .menu-open').hide();
+          $('.navigation-wrapper').addClass('mobile-navigation');
+          $('.navbar').addClass('menu-open');
+          $('.menu-close').show();
+        });
+        $('.menu-close').on('click', function(){
+          $('.navigation-wrapper').removeClass('mobile-navigation');
+          $('.navbar').removeClass('menu-open');
+          $('.menu-close').hide();
+        });
     
+        var active_url = "";
     
-</style>
-<script type="text/javascript" src="/static/js/horizon-swiper.min.js"></script>
-<script type="text/javascript">
-  $(document).ready( function(){
-    // if($('#scrollable-pane li').length > 3){
-      $('.horizon-swiper').horizonSwiper({
-        showItems: 3,
-        arrows: true
-      }); 
-    // }
+        $(".navbar-mobile-header-icon").click(function(index){
+          $('.navigation-wrapper').toggleClass('menu-overlay');
+          $(this).toggleClass("navbar-mobile-header-icon-click navbar-mobile-header-icon-out");
+          $(".mobile-inner-nav").slideToggle(500);
+          $(".mobile-inner-nav .navbar-nav a").each(function( index ) {
+            $(this).css({'animation-delay': (index/25)+'0.4s'});
+          });
+        });
     
-    $('.navbar-toggle').on('click', function(){
-      $('.mobile-navigation, .menu-open').hide();
-      $('.navigation-wrapper').addClass('mobile-navigation');
-      $('.navbar').addClass('menu-open');
-      $('.menu-close').show();
-    });
-    $('.menu-close').on('click', function(){
-      $('.navigation-wrapper').removeClass('mobile-navigation');
-      $('.navbar').removeClass('menu-open');
-      $('.menu-close').hide();
-    });
-
-    var active_url = "";
-
-    $(".navbar-mobile-header-icon").click(function(index){
-      $('.navigation-wrapper').toggleClass('menu-overlay');
-      $(this).toggleClass("navbar-mobile-header-icon-click navbar-mobile-header-icon-out");
-      $(".mobile-inner-nav").slideToggle(500);
-      $(".mobile-inner-nav .navbar-nav a").each(function( index ) {
-        $(this).css({'animation-delay': (index/25)+'0.4s'});
-      });
-    });
-
-    $('.dropdown-btn').on('click', function(e){
-      e.preventDefault();
-      setDropdown();
-    });
+        $('.dropdown-btn').on('click', function(e){
+          e.preventDefault();
+          setDropdown();
+        });
+        
+        function setDropdown() {
+          $('#myDropdown').slideToggle();
+        }
     
-    function setDropdown() {
-      $('#myDropdown').slideToggle();
-    }
-
-    $('body').on('click', function(event){
-      if (!event.target.matches('.dropdown-btn')) {
-
-        var dropdowns = $(".dropdown-content");
-        var i;
-        for (i = 0; i < dropdowns.length; i++) {
-          var openDropdown = dropdowns[i];
-          if (openDropdown.classList.contains('show')) {
-            openDropdown.classList.remove('show');
+        $('body').on('click', function(event){
```

Comment:
These should probably also move to docs.js but they can wait.

Reply:
++ I'll plan on moving this later this week.